src/controllers/tracker.controller.js

```diff
@@ -0,0 +1,13 @@
+import commandMap from "./commands/commandMap.js";
+
```

Comment:
check for understanding: Is it common to name files like this?
tracker.controller.js vs just tracker.js

Reply:
answered above

Reply:
answered above